Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,12 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+const wholeNumberPart = math.floor(num);
+const decimalPart = num - wholeNumberPart;
+const roundedNum = math.round(num);
+
+console.log(`whole number part: ${wholeNumberPart}`);
+console.log(`decimal part: ${decimalPart}`);
```

Comment:
If you run the program, you may notice that console.log() outputs the value of `decimalPart` with more than 4 decimal places (which is normal because of floating point arithmetic). 
If you were asked to print the value of `decimalPart` to exactly 4 decimal places, how would you modify the code?